metaserver/src/main/scala/scala/meta/languageserver/search/SymbolIndex.scala

```diff
@@ -60,13 +60,11 @@ class SymbolIndex(
     for {
       document <- documentIndex.getDocument(path.toNIO.toUri)
       _ = logger.info(s"Found document for $path")
-      _ <- isFreshSemanticdb(path, document)
```

Comment:
🎉

Reply:
hehe guess that one wasn't very popular 😅

Reply:
@olafurpg just check https://github.com/laughedelic/atom-ide-scala/issues/20 and https://github.com/laughedelic/atom-ide-scala/pull/22 😉

Reply:
:joy: @laughedelic